scripts/inference/run_mpt_with_ft.py

```diff
@@ -280,7 +280,7 @@ def main():
     shared_contexts_ratio = args.shared_contexts_ratio
     layernorm_eps = args.layernorm_eps
     use_attention_linear_bias = args.alibi
-    has_positional_encoding = not args.alibi
+    has_positional_encoding = not args.alibi  # TODO: Should probably be: has_positional_encoding = not (args.alibi or args.rope)
```

Comment:
are we leaving this as a todo? or asking @dskhudia to comment on what should be done here?

Reply:
I checked, this should actually be `not args.alibi`. This is because this argument is (only) used for instantiating [ParallelGPT](https://github.com/NVIDIA/FasterTransformer/blob/df4a7534860137e060e18d2ebf019906120ea204/examples/pytorch/gpt/utils/gpt.py#L441), which seems to only support learned positional encodings or attention linear biases (alibi).

Reply:
So the added `TODO` should be removed?

Reply:
Yes, removed it in the latest commit.